jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/skaffold/FilesMojoV2.java

```diff
@@ -183,7 +183,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     String property = newProperty != null ? newProperty : deprecatedProperty;
     if (property != null) {
       List<String> paths = ConfigurationPropertyValidator.parseListProperty(property);
-      return paths.stream().map(Paths::get).collect(Collectors.toList());
+      return paths.stream().map(Paths::get).map(Path::toAbsolutePath).collect(Collectors.toList());
```

Comment:
I think this is not enough. For example, users can currently set the extra path to a project-relative path (like `src/main/my-custom`), and this will just become `/src/main/my-custom`. I *think* what should be done is to resolve this against the project root in case it is a relative path. @briandealwis is that right?

Reply:
Yeah I just realized the problem as I was updating tests. We have to check if they're absolute, and otherwise resolve against the current module root.